src/main/java/net/openhft/chronicle/hash/impl/stage/entry/SegmentStages.java

```diff
@@ -113,7 +113,7 @@
 
     @NotNull
     private static VanillaBytes unmonitoredVanillaBytes(PointerBytesStore segmentBS) {
-        VanillaBytes bytes = new VanillaBytes(segmentBS);
+        VanillaBytes bytes = new VanillaBytes(segmentBS) {};
```

Comment:
Why not VanillaBytes.vanillaBytes()?

Reply:
It returns NativeBytes, not sure what to do with that.